cmsimple/functions.php

```diff
@@ -2369,6 +2369,25 @@ function XH_registeredEditmenuPlugins()
     return XH_registerPluginType('editmenu');
 }
 
+/**
+ * Returns the names of the registered languagemenu plugins.
+ *
+ * @return string
+ *
+ * @since 1.7.6
```

Comment:
I'd prefer
```suggestion
 * @since 1.8.0
```
```suggestion
 * @since 1.7.6
```

Reply:
I am confused.
Which is it 1.7.6 or 1.8?
And if 1.8, why?
Everything works as before, no BC break. There is no reason to postpone this.

Reply:
I don't think there should be a 1.7.6 at all (unless for security fixes). We really should move ahead and do 1.8.0 (even if a lot of features would need to be postponed to 1.9.0 or later).

And generally, I would try to follow [SemVer](https://semver.org/lang/de/) – no new features in a revision.